client-lite/src/trace/do_log.h

```diff
@@ -1,24 +1,36 @@
 #pragma once
 
-void DoTraceLoggingRegister();
-void DoTraceLoggingUnregister();
+#include <string>
 
-bool IsLoggingEnabled(boost::log::trivial::severity_level level);
-bool IsVerboseLoggingEnabled();
+namespace DOLog
+{
 
-void LogRuntimeFailure(const docli::FailureInfo& failure) noexcept;
+enum class Level
+{
+    Error,
+    Warning,
+    Info,
+    Verbose,
```

Comment:
>, [](http://example.com/codeflow?start=11&length=1)

nit: does last one need comma?

Reply:
No but it's a convention to allow cleaner diff when adding a new entry - no need to modify existing lines of code.